developer_scripts/import.py

```diff
@@ -0,0 +1,67 @@
+import xlsxwriter
```

Comment:
Could you add a shebang line `#!/usr/bin/env python` here for convenience?

Reply:
Any scripts should be packaged correctly http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html